packages/ibm-products/src/components/GetStarted/GetStarted.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright IBM Corp. 2024, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import { getDevtoolsProps } from '../../global/js/utils/devtools';
+import { pkg } from '../../settings';
+
+import { Card } from '../Card';
+const componentName = 'GetStarted';
+
+/**
+ * GetStarted a card with icon, number, and media variants
+ */
+export let GetStarted = React.forwardRef(({ ...rest }, ref) => {
+  return (
+    <Card getStarted ref={ref} {...rest} {...getDevtoolsProps(componentName)} />
```

Comment:
I saw your comment that we used the internal `Card` instead of `ProductiveCard` — were we unable to get the actions working using `ProductiveCard`?

I was thinking originally something like ...

```suggestion
    <ProductiveCard ref={ref} {...rest} {...getDevtoolsProps(componentName)}>
      <div className={`${blockClass}__header`}>
        {sequence && (
          <div className={`${blockClass}__sequence`}>{sequence}</div>
        )}
        {status && (
          <div className={`${blockClass}__status`}>
            ...
          </div>
        )}
      </div>
      <div className={`${blockClass}__footer`}>
        <div className={`${blockClass}__metadata`}>
          ...
        </div>
      </div>
    </ProductiveCard>
```

... but I’m guessing this was a bit more complicated because of the positioning of sequence/status before the title/label and other media elements. 🤔

Still, anything we can keep specific to just the `GetStarted` component will likely help reduce complexity at the internal `Card` level longer term.